README.md

```diff
@@ -190,27 +190,26 @@ If a track implements an exercise for which test data exists, the exercise _must
+[19709124-b82e-4e86-a722-9e5c5ebf3952]
+description = "no name given"
 
-# no name given
-"19709124-b82e-4e86-a722-9e5c5ebf3952" = true
+[3451eebd-123f-4256-b667-7b109affce32]
+description = "a name given"
+include = false
 
-# a name given
-"3451eebd-123f-4256-b667-7b109affce32" = true
-
-# another name given
-"653611c6-be9f-4935-ab42-978e25fe9a10" = false
+[653611c6-be9f-4935-ab42-978e25fe9a10]
+description = "another name given"
 ```
 
-In this case, the track has chosen to implement two of the three available tests.
+In this case, the track has chosen to implement two of the three available tests (`include = true` is the default and is omitted).
 
 If a track uses a _test generator_ to generate an exercise's test suite, it _must_ use the contents of the `tests.toml` file to determine which tests to include in the generated test suite.
 
 ### Track Test Data Tooling
 
-To make it easy to keep the `tests.toml` up to date, tracks can use the [`canonical_data_syncer` application](https://github.com/exercism/canonical-data-syncer). This application is a small, standalone binary that will compare the tests specified in the `tests.toml` files against the tests that are defined in the exercise's canonical data. It then interactively gives the maintainer the option to include or exclude test cases that are currently missing, updating the `tests.toml` file accordingly.
+To make it easy to keep the `tests.toml` up to date, tracks can use the [`configlet` application](https://github.com/exercism/configlet)'s `sync` command. This command will compare the tests specified in the `tests.toml` files against the tests that are defined in the exercise's canonical data. It then interactively gives the maintainer the option to include or exclude test cases that are currently missing, updating the `tests.toml` file accordingly.
 
-To use the canonical data syncer tool, tracks should copying the [`fetch-canonical_data_syncer`](https://github.com/exercism/canonical-data-syncer/blob/main/scripts/fetch-canonical_data_syncer) and/or [`fetch-canonical_data_syncer.ps1`](https://github.com/exercism/canonical-data-syncer/blob/main/scripts/fetch-canonical_data_syncer.ps1) scripts into their repository. Then, running either of these scripts will download the latest version of the tool to the track's `bin` directory. The tool can be run using `./bin/canonical_data_syncer` or `.\bin\canonical_data_syncer.exe`, depending on your operating system.
+To use the configlet tool, tracks should copy the [`fetch-configlet`](https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet) and/or [`fetch-configlet.ps1`](https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet.ps1) scripts into their repository's `bin` directory. Then, running either of these scripts will download the latest version of the tool to the same `bin` directory. The tool can then be run using `./bin/configlet sync` or `.\bin\configlet.exe sync`, depending on your operating system.
```

Comment:
```suggestion
To download the latest version of the `configlet` tool, please run the [`fetch-configlet`](https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet) script or the [`fetch-configlet.ps1`](https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet.ps1) script. At least one of these scripts should already exist in every track repo's `bin` directory - the script will also download `configlet` to this location. You can then sync the tests by running `./bin/configlet sync` (on Linux or macOS) or `.\bin\configlet.exe sync` (on Windows).
```

Non-blocking suggestion.

This suggested wording could be improved, but the idea is that it's better to tell the user to run the one that's already in the track repo. If we tell them to download the fetch script themselves, the user still has to save it to the `bin` directory - most of the time this just means they'll be prompted about overwriting the existing one.

Referring to `configlet sync` as "the tool" also felt a bit awkward - I'd say that "the tool" is `configlet` itself.